### PR TITLE
DP-29882: Fix plain node links

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-29882

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6deab6acb5d58b489f2cb9fbbc2745de6d4652e6
</p>

File: composer.json [hunk 1]
```diff
@@ -213,7 +213,7 @@
         "drupal/key": "^1.10",
         "drupal/key_auth": "^2.1",
         "drupal/key_value_field": "^1",
-        "drupal/linkit": "^6",
+        "drupal/linkit": "^6.0",
```
Comment (weitzman) [inline composer.json:216]: See Jira ticket. I dont think this PR will be needed once we put back the Pathologic patch we have always used.

**Review by weitzman — CHANGES_REQUESTED** [review]

**Review by weitzman — APPROVED** [review]

**Comment by weitzman** [comment]
Does the new LinkIt patch apply cleanly to Linkit 6.1.1? If not, we should make another patch for 6.1.1 so we dont block that PR. The D10 PR is currently using the https://www.drupal.org/project/linkit/issues/2877535#comment-15255425 patch

**Comment by arthurbaghdas** [comment]
> Does the new LinkIt patch apply cleanly to Linkit 6.1.1? If not, we should make another patch for 6.1.1 so we dont block that PR. The D10 PR is currently using the https://www.drupal.org/project/linkit/issues/2877535#comment-15255425 patch

Why? We can stick to version 6.0.2 which is compatible with drupal core 10.1 and don't waste our time now on refactoring or even re-architecting the patch. I'll see if this applies and works on D10 branch, if not I'll revert the version to 6.0.2 to avoid wasting time now.

**Comment by arthurbaghdas** [comment]
> > Does the new LinkIt patch apply cleanly to Linkit 6.1.1? If not, we should make another patch for 6.1.1 so we dont block that PR. The D10 PR is currently using the https://www.drupal.org/project/linkit/issues/2877535#comment-15255425 patch
> 
> Why? We can stick to version 6.0.2 which is compatible with drupal core 10.1 and don't waste our time now on refactoring or even re-architecting the patch. I'll see if this applies and works on D10 branch, if not I'll revert the version to 6.0.2 to avoid wasting time now.

In fact, just tried, it cleanly applies to the latest 6.1.x version and I've replaced the patch in the D10 PR as well, so we should be good to go with this one here.

**Pull request merged**